.github/workflows/merge_queue.yml

```
@@ -0,0 +1,869 @@
+    types: [checks_requested]
+
+permissions:
+  pull-requests: write
+  id-token: write
+  contents: write
+  statuses: write
+
+jobs:
+  context:
+    name: Context
+    if: ${{ needs.access.outputs.granted == 'true' }}
```

actionlint (1.7.4)

property "access" is not defined in object type {}

(expression)